### PR TITLE
bugfix and test extents

### PR DESCRIPTION
**Pull request by rafaqz**
We had missed points, and not correctly implemented the interface. And not tested! 

The extent tests were all for the whole shape file. Anyway, fixed.

### CODE REVIEW

File: src/extent.jl [hunk 1]
```diff
@@ -14,12 +14,20 @@ GI.ismeasured(::GI.AbstractPointTrait, ::Point) = false
 GI.ismeasured(::GI.AbstractPointTrait, ::Union{PointM,PointZ}) = true
 
 # extent
-function GI.extent(x::AbstractShape)
+GI.extent(::GI.PointTrait, p::Union{Point,PointM}) =
+    Extents.Extent(X=(p.x, p.x), Y=(p.y, p.y))
+GI.extent(::GI.PointTrait, p::PointZ) =
+    Extents.Extent(X=(p.x, p.x), Y=(p.y, p.y), Z=(p.y, p.y))
```
Comment (visr) [inline src/extent.jl:20]:
```suggestion
    Extents.Extent(X=(p.x, p.x), Y=(p.y, p.y), Z=(p.z, p.z))
```

**Review by visr — CHANGES_REQUESTED** [review]

File: src/extent.jl (hunk 1 above)
Comment (rafaqz) [reply src/extent.jl:20]: lol, you can tell this was slightly rushed

**Comment by rafaqz** [comment]
I guess it needs a few exact tests after that error... I'll add some.

File: src/extent.jl (hunk 1 above)
Comment (rafaqz) [reply src/extent.jl]: It has a test now

**Comment by rafaqz** [comment]
Had to get this merged its breaking CI elsewhere

**Pull request merged**